components/gitpod-protocol/src/util/logging.ts

```diff
@@ -246,7 +247,16 @@ namespace GoogleLogSeverity {
     };
 }
 
+const logsCounter = new prometheusClient.Counter({
+    name: "gitpod_logs_total",
+    help: "Total number of logs by level",
+    labelNames: ["level"],
+    registers: [prometheusClient.register],
+});
+
 function doLog(calledViaConsole: boolean, consoleLog: ConsoleLog, severity: GoogleLogSeverity, args: unknown[]): void {
+    logsCounter.labels(severity).inc();
```

Comment:
level here is `type GoogleLogSeverity = "EMERGENCY" | "ALERT" | "CRITICAL" | "ERROR" | "WARNING" | "INFO" | "DEBUG";`